### DISCUSSION TITLE
Case insensitive search in Relation

### DISCUSSION THREAD

**Discussion by ghost** [orchidsoftware/platform#1832] [Q&A]
How to make subj?

![img2](https://user-images.githubusercontent.com/56771968/129690866-fb4c0384-12af-4911-b900-39508898bfe4.png)
![img1](https://user-images.githubusercontent.com/56771968/129690871-70c3d44e-fb48-421f-b349-d608c52af463.png)

**Comment by v-joel** [comment]
You can try editing the file vendor/orchid/platform/src/Filters/HttpFilter.php and changing the filtersExact function.

Change the line
return $query->where($property, 'like', "%$value%");
to
return $query->where($property, 'ilike', "%$value%");

**Reply by ghost** [reply]
Hello!
i dont want to modify vendor files :)
